tests/test_api_build.py

```diff
@@ -132,11 +121,12 @@ def test_ignore_prefix_files(testing_config, monkeypatch):
 # Regardless of the reason for skipping, we should definitely find a better way for tests to look for the packages
 # Rather than assuming they will be at $ROOT/pkgs since that can change and we don't care where they are in terms of the
 # tests.
+# Need more time to figure the problem circumventing..
 def test_ignore_some_prefix_files(testing_config, monkeypatch):
     recipe = os.path.join(metadata_dir, "_ignore_some_prefix_files")
     testing_config.activate = True
     monkeypatch.setenv("CONDA_TEST_VAR", "conda_test")
-    monkeypatch.setenv("CONDA_TEST_VAR_2", "conda_test_2")
+    monkeypatch.setenv("CONDA_TEST_VAR_3", "conda_test_3")
```

Comment:
Is there a reason to change this from CONDA_TEST_VAR_2 to CONDA_TEST_VAR_3?

Reply:
after reviewing `CONDA_TEST_VAR`, `CONDA_TEST_VAR_2`, and `CONDA_TEST_VAR_3` usage I actually don't think they're used in most cases, `CONDA_TEST_VAR` and `CONDA_TEST_VAR_2` are only referenced in `tests/test-recipes/metadata/jinja_vars`